JSDemos/Demos/Charts/ServerSideDataProcessing/description.md

```diff
@@ -1 +1,9 @@
-In&nbsp;many cases, before being handed out to&nbsp;the client, data is&nbsp;processed on&nbsp;the server. The DevExtreme JavaScript UI components support this scenario. In&nbsp;this demo, the data source of&nbsp;the Chart component loads weather data for a&nbsp;selected month from an&nbsp;OData service. Each time the month is&nbsp;changed, the data source sends a&nbsp;new query to&nbsp;the service.
+In many cases, you need to process data on the server before the chart displays it. The Chart component supports this scenario. 
```

Comment:
> In many cases, you need to process data on the server before **the chart** displays it.

I have a question: by "the chart", do you mean the Chart component or just **any** chart? In the latter case, you've chosen an incorrect article; otherwise, you should have capitalized "chart".

Reply:
Yeah, I made a mistake with the article. I wanted to said "a chart" like "visual element" in the first sentence, and then talk about the Chart component in the second.

Reply:
OK. I thought as much.